README.md

```diff
@@ -18,46 +18,53 @@ Monitored is a wrapper function that writes success/error logs and [StatsD](http
 ## Quick start
 
 ### Yarn
+
 ```bash
 yarn add monitored
 ```
+
 ### Npm
+
 ```bash
 npm install --save monitored
 ```
 
 <br>
 
 ## Initialize
+
 ### Call `setGlobalInstance` at the root of the project
 
 To wire this package, you need to pass an `Options` object.
 
-* `serviceName` — Represents the name of the service you are monitoring (mandatory)
-* `logger` — Writes success and error logs with the passed in logger (optional)
-* `statsd` — Writes metrics to StatsD server (optional)
-* `mock` — Writes the metrics to logs instead of StatsD for debugging. defaults to false (optional)
-* `shouldMonitorExecutionStart` — When true will log execution start and will increment a metrics. defaults to true (optional)
-* `disableSuccessLogs` — When true, will not send success log. defaults to false (optional)
-<br>
+-   `serviceName` — Represents the name of the service you are monitoring (mandatory)
+-   `logger` — Writes success and error logs with the passed in logger (optional)
```

Comment:
The description does not match the actual options passed

Reply:
@gioragutt Can you contribute a description?

Reply:
updated. @gioragutt wdyt?